activerecord/test/cases/migration/columns_test.rb

```diff
@@ -142,6 +142,7 @@ def test_remove_column_with_index
       end
 
       def test_remove_column_with_multi_column_index
+        skip if current_adapter?(:Mysql2Adapter) && @connection.mariadb? && @connection.version >= "10.2.8"
```

Comment:
Can you add a comment on the line why we need to skip for the MariaDB version?

Reply:
Comment added and squashed.